index.html

```diff
@@ -499,8 +499,12 @@ <h1>Terminology</h1>
       <dt>
         <dfn data-lt="WoT Discoverer" class="lint-ignore">Discoverer</dfn>
       </dt>
-      <dd>An entity that generates and registers a TD with an exploration service
-        on behalf of a <a>Thing</a> which may not be able to do it for itself.
+      <dd>An entity which acts as a client of a WoT Discovery service to discover 
```

Comment:
The first sentence is a bit misleading and gives the impression that discoverer cannot do the exploration mechanisms?

Reply:
Since the discovery service generally means the TDD no?

Reply:
Arch call on May 11th:
Change: WoT Discovery service -> WoT Discovery process